ipatests/test_integration/test_adtrust_install.py

```diff
@@ -0,0 +1,190 @@
+#
+# Copyright (C) 2020  FreeIPA Contributors see COPYING for license
+#
+
+"""This module provides tests for ipa-adtrust-install utility"""
+
+import re
+import textwrap
+
+from ipaplatform.paths import paths
+from ipapython.dn import DN
+from ipatests.pytest_ipa.integration import tasks
+from ipatests.test_integration.base import IntegrationTest
+
+
+class TestIpaAdTrustInstall(IntegrationTest):
+    topology = 'line'
+    num_replicas = 1
+
+    def unconfigure_replica_as_agent(self, host):
+        """ Remove a replica from the list of agents.
+
+        cn=adtrust agents,cn=sysaccounts,cn=etc,$BASEDN contains a list
+        of members representing the agents. Remove the replica principal
+        from this list.
+        """
+        remove_agent_ldif = textwrap.dedent("""
```

Comment:
host has an ldap_connect method which returns LDAPClient object which allows to do LDAP modifications in a more elegant way without formatted diffs. Can it be used here?

Reply:
The code is more compact with tasks.ldapmodify_dm IMO.